### DISCUSSION TITLE
Capstone ML - F&B Time Series - Error Evaluation & Submission

### DISCUSSION THREAD

**Discussion by tu849035** [teamalgoritma/community#348] [Q&A] [answered]
Salam.

Terjadi error seperti tampilan di bawah saat saya mencoba melakukan submission.
![image](https://user-images.githubusercontent.com/107348211/192140214-dc416642-4e50-4b48-8fb4-34d178fb35d5.png)

Mohon bantuan saran & solusinya.

**Comment by fafilia** [comment, marked as answer]
Dear @tu849035 ,

Error tersebut terjadi karena nilai yang akan Anda coba mutate ke sebuah kolom masih berupa object forecast. Solusinya, bisa dicoba untuk memanggil atribut mean yang ada pada object forecast seperti contoh berikut:
```
mutate(visitor = forecast_object$mean)
```

Demikian yang dapat saya sampaikan, semoga bermanfaat.

**Reply by tu849035** [reply]
Baik mbak terima kasih.
Sudah saya coba. Tapi hasil accuracynya tidak memenuhi syarat. apakah ada atribut lain yg bisa diambil selain `mean`?

Mohon bantuannya.

**Reply by fafilia** [reply]
atribut mean tersebut dimaksudkan untuk menunjukkan nilai prediksi/forecastnya saja Pak. Jika dari nilai-nilai tersebut menghasilkan evaluasi yang masih kurang, artinya Bapak harus melakukan tunning terhadap modelnya. Sebagai saran, disini Bapak bisa membuat lebih dari 1 model untuk nantinya dibandingkan hasil forecastnya.
